Tasks/MavenAuthenticateV0/taskJsonOverride.json

```diff
@@ -0,0 +1,35 @@
+{
+    "inputs": [
+      {
+        "name": "workloadIdentityServiceConnection",
+        "label": "'Entra Workload ID-backed Azure DevOps user' Service Connection",
+        "helpMarkDown": "This is an Entra Workload ID-backed Azure DevOps user Service Connection. If this is set, feedUrl is required. All other inputs are ignored.",
```

Comment:
Remove feedUrl is required

Reply:
Update this string to something that explains the current behavior

Reply:
Changed this and the description for artifactsFeeds